pageflow.gemspec

```diff
@@ -9,8 +9,10 @@ Gem::Specification.new do |s|
   s.email       = ['info@codevise.de']
   s.homepage    = 'http://www.pageflow.io'
   s.summary     = 'Multimedia story telling for the web.'
+  s.license     = 'MIT'
 
   s.files = Dir['{admins,app,config,db,lib,vendor,spec/factories,spec/fixtures}/**/*', 'MIT-LICENSE', 'Rakefile', 'README.md', 'CHANGELOG.md']
+  s.require_paths = ['lib']
```

Comment:
This appears to [be the default](http://guides.rubygems.org/specification-reference/#require_paths=). But being explicit can't hurt.

Reply:
I agree. I tend to include it nowadays, so developers are exposed to it. It can explain some weird gem behaviour for example.